### PR TITLE
docs: Point users to react-fontawesome for icons

### LINKED ISSUE #84: Documentation on Icons
Since USWDS recommends using FontAwesome for icons (https://designsystem.digital.gov/components/icons/), it doesn't seem necessary to create an Icon component. It might make more sense to officially recommend using https://github.com/FortAwesome/react-fontawesome and provide documentation on how to use it in your project.

### PR DESCRIPTION
**Pull request by jim**
# Summary

This PR adds documentation to the README that recommends the use of `react-fontawesome` for projects that need icons.

I'm not sure if/what should be added to the example app. I put a page in there to make sure that the icons worked, but I can take that out if it doesn't make sense to keep since it is testing an external dependency.

## Related Issues or PRs

closes #84 

## How To Test

* Read through the *Icons* section of the docs for clarity.
* Open the example app locally and look at the *Icons* page.

### Screenshots

Here is the new page in the example app:

Screen Shot 2020-07-29 at 3 18 52 PM

## Checklist

- [ ] PR title conforms to conventional commits, e.g. feat: Button
- [ ] PR includes `yarn audit` output if dependencies changed
- [ ] Any new components are exported from `index.ts`

### CODE REVIEW

**Comment by trussworks-infra** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>This PR does not include changes to storybook, even though it affects component code.</td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>This PR does not include changes to tests, even though it affects source code.</td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 47263b8f573b92660388165f3cf89247ac0072f1
</p>

File: example/src/pages/Icons.tsx [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import { faUpload } from '@fortawesome/free-solid-svg-icons'
+
+const IconsPage = (): React.ReactElement => (
+  <section>
+    <h1>Icons</h1>
+
+    <p className="usa-intro">
+      We recommend using icons from <a href="https://www.google.com/search?client=safari&rls=en&q=react-fontawesome&ie=UTF-8&oe=UTF-8">react-fontawesome</a> for projects that need icons.
```
Comment (suzubara) [inline example/src/pages/Icons.tsx:12]: did you mean to link to the google search results?

**Comment by haworku** [comment]
To respond to your @jim comment in the PR description - I like how you added it to the example page.  It doesn't hurt to remind folks a few ways this exists.  thanks!

File: example/src/pages/Icons.tsx (hunk 1 above)
Comment (jim) [reply example/src/pages/Icons.tsx:12]: LOL nope!

**Pull request merged**